### PR TITLE
fix(components/tabs): vertical tab animation states reset correctly when transitioning between responsive states

### PR DESCRIPTION
**Pull request by Blackbaud-TrevorBurch**

### CODE REVIEW

**Comment by blackbaud-sky-build-user** [comment]
[Storybook preview](https://blackbaud.github.io/skyux-pr-preview/1282/storybook/)

Component Storybooks:

- [a11y-storybook](https://blackbaud.github.io/skyux-pr-preview/1282/storybooks/a11y-storybook/)
- [ag-grid-storybook](https://blackbaud.github.io/skyux-pr-preview/1282/storybooks/ag-grid-storybook/)
- [core-storybook](https://blackbaud.github.io/skyux-pr-preview/1282/storybooks/core-storybook/)
- [indicators-storybook](https://blackbaud.github.io/skyux-pr-preview/1282/storybooks/indicators-storybook/)
- [layout-storybook](https://blackbaud.github.io/skyux-pr-preview/1282/storybooks/layout-storybook/)
- [lists-storybook](https://blackbaud.github.io/skyux-pr-preview/1282/storybooks/lists-storybook/)
- [modals-storybook](https://blackbaud.github.io/skyux-pr-preview/1282/storybooks/modals-storybook/)
- [pages-storybook](https://blackbaud.github.io/skyux-pr-preview/1282/storybooks/pages-storybook/)
- [popovers-storybook](https://blackbaud.github.io/skyux-pr-preview/1282/storybooks/popovers-storybook/)
- [split-view-storybook](https://blackbaud.github.io/skyux-pr-preview/1282/storybooks/split-view-storybook/)
- [storybook](https://blackbaud.github.io/skyux-pr-preview/1282/storybooks/storybook/)
- [tabs-storybook](https://blackbaud.github.io/skyux-pr-preview/1282/storybooks/tabs-storybook/)
- [theme-storybook](https://blackbaud.github.io/skyux-pr-preview/1282/storybooks/theme-storybook/)
- [tiles-storybook](https://blackbaud.github.io/skyux-pr-preview/1282/storybooks/tiles-storybook/)

Apps:

- [code-examples](https://blackbaud.github.io/skyux-pr-preview/1282/code-examples/)
- [dep-graph](https://blackbaud.github.io/skyux-pr-preview/1282/dep-graph/)
- [integration](https://blackbaud.github.io/skyux-pr-preview/1282/integration/)
- [playground](https://blackbaud.github.io/skyux-pr-preview/1282/playground/)

File: libs/components/tabs/src/lib/modules/vertical-tabset/vertical-tabset-group.component.ts [hunk 1]
```diff
@@ -81,11 +99,17 @@ export class SkyVerticalTabsetGroupComponent implements OnInit, OnDestroy {
     this.#ngUnsubscribe.complete();
   }
 
+  public updateSlideDirection(event: any): void {
+    console.log(event);
```
Comment (Blackbaud-ErikaMcVey) [inline libs/components/tabs/src/lib/modules/vertical-tabset/vertical-tabset-group.component.ts]: straggler console.log

File: libs/components/tabs/src/lib/modules/vertical-tabset/vertical-tabset-group.component.ts [hunk 2]
```diff
@@ -98,14 +122,20 @@ export class SkyVerticalTabsetGroupComponent implements OnInit, OnDestroy {
...
 
   public tabsShown = () => {
-    this.open = this.#openBeforeTabsHidden;
+    // Set the animation back up so that the "void" state is returned to where it was prior to the tabs being hidden.
+    // This will be instantaneous due to there not being a "void -> *" state on the slide animation.
+    this.#slideForExpanded(true);
     this.#changeRef.markForCheck();
   };
+
+  #slideForExpanded(animate: boolean): void {
```
Comment (Blackbaud-ErikaMcVey) [inline libs/components/tabs/src/lib/modules/vertical-tabset/vertical-tabset-group.component.ts]: the naming of this is confusing me a little. Is it specifically setting animation related properties only when the tabs are expanded? Is it triggering the animation?

**Review by Blackbaud-ErikaMcVey — APPROVED** [review]

**Comment by blackbaud-sky-build-user** [comment]
Cherry-pick successful! :cherries: :tada: See https://github.com/blackbaud/skyux/pull/1289

**Pull request merged**